build.xml

```diff
@@ -421,15 +421,36 @@
         </condition>
         <echo>Using output folder: ${output.folder}</echo>
 
+        <!-- if old.default is picked, check if available, otherwise get -->
+        <condition property="old.default-and-available" value="true">
+            <and>
+                <equals arg1="${old.file}" arg2="${old.default}"/>
+                <available file="${old.file}"/>
+            </and>
+        </condition>
+
+        <get src="https://raw.githubusercontent.com/music-encoding/schema/main/4.0.1/mei-source_canonicalized_v4.0.1.xml" dest="${output.folder}${old.default}" unless:true="${old.default-and-available}"/>
+
+        <!-- check if old.file is available -->
+        <condition property="old.available" value="true" else="false">
+            <or>
+              <available file="${old.file}"/>
+              <available file="${output.folder}${old.default}"/>
```

Comment:
this actually is not so good because $old.available might return true although $olf.file is not set to $old.default. It would be better to add the path directly to $old.default. That again would need $output.folder to be defined earlier. (see above)

Reply:
cannot comment on unchanged lines but $output.folder would have to be defined above old.default. then again, why not put download the old.default to build or temp?